### PR TITLE
Support volume backup service with snapshot

### PR DESCRIPTION
**Pull request by AmitRoushan**
**What type of PR is this?**
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
>
> /kind new feature
> /kind bug fix
> /kind cleanup
> /kind revert change
> /kind design
> /kind documentation
> /kind enhancement

/kind enhancement

**What this PR does / why we need it**:
Added support for Volume Snapshot before volume backup 

**Which issue(s) this PR fixes**:

Fixes #

**Test Report Added?**:
> Uncomment only one ` /kind <>` line, hit enter to put that in a new line, and remove leading whitespace from that line:
> /kind TESTED
> /kind NOT-TESTED

**Test Report**:

**Special notes for your reviewer**:

Signed-off-by: Amit Roushan <amitroushan.cusat11@gmail.com>

### CODE REVIEW

**Review by joseph-v — APPROVED** [review]

**Review by PravinRanjan10 — APPROVED** [review]
LGTM

File: controllers/backup/backupvolume.go [hunk 1]
```diff
@@ -159,71 +170,27 @@ func (ctrl *controller) removeVolumeBackup(
...
-		k8sPV.DeepCopyInto(&pv)
-		pvs = append(pvs, pv)
-		pvNames.Insert(pv.Name)
 	}
 
-	return pvs, nil
+	return pvcs, nil
 }
 
 func (ctrl *controller) ensureVolumeBackupContent(
```
Comment (SrinivasChilveri) [inline controllers/backup/backupvolume.go:196]: I feel we can remove this function which is not at all used with the changes

**Comment by SrinivasChilveri** [comment]
lgtm

**Review by sushanthakumar — APPROVED** [review]
LGTM

**Pull request merged**